src/components/calcite-input/calcite-input.tsx

```diff
@@ -169,15 +174,58 @@ export class CalciteInput {
     this.stepString = this.step?.toString();
     this.slottedActionEl = this.el.querySelector("[slot=input-action]");
     if (this.disabled) this.setDisabledAction();
+    if (this.type === "textarea") setTimeout(() => (this.value = this.slotValue));
   }
 
+  // componentDidRender(): void {
+  //   this.labelValue = this.value || this.placeholder;
+  // }
+
```

Comment:
I can't for the life of me figure out why evaluating `this.value || this.placeholder` does not return the correct string when rendering the inline editable label for a textarea. It works just fine for text inputs. The only way i can get around it is be uncommenting this `componentDidRender` hook to force a re-render. This is of course not optimal, and I would ❤️  some help figuring out what might be causing the boolean operation to fail. I can render `this.value` without any problems, but we need to be able to show the placeholder value in the label when the user clears the value.

Reply:
Can this check be moved into the render method?

Reply:
The current (non-working version) is evaluating that boolean expression in the render method. By uncommenting this hook, setting the `this.labelValue` state value causes an extra re-render which makes it work like we want.

Reply:
As a note... we hope to split textarea out into a separate component at some point soon (https://github.com/Esri/calcite-components/issues/863). Doesn't help at this point but as a general note :)

Reply:
I was able to get it to work when setting it in "componentDidLoad", and then a check again on "componentDidUpdate", using an internal prop vs. state (and removing the "get labelStatus"). Unsure if this would have other unintended results with the feature but... seemed to work.

Reply:
https://jmp.sh/dH9aly3+/Screencast+2020-10-22+at+10.50.05+AM.mp4 : "will load" only called once, "did update" called on each update as expected, I think that works? I think maybe using State causes re-render as that changes perhaps?

Reply:
Did you happen to check the textarea demo? I guess I wasn't clear, but the bug is only present for textareas. Seems to work fine for text inputs. Making it that much more of a mystery.